dist/bin/eslint.config.js

```diff
@@ -0,0 +1,26 @@
+const iTwinPlugin = require("../rules/index");
```

Comment:
I'm not a fan of this import. The "plugin" should be defined in the main barrel file, not in a subdirectory. What if I want to use this plugin for its rules, but without extending one of its configs? How would I import the plugin into my external package?

Reply:
changed require here, going for the main index file works just as well